code/API_definitions/location-retrieval.yaml

```diff
@@ -270,7 +268,24 @@ components:
               type: number
               description: Distance from the center in meters
               minimum: 1
-        
+
+    Polygon:
+      allOf:
+        - $ref: "#/components/schemas/Area"
+        - type: object
+          required:
+            - pointList
+          properties:
+            pointList:

```

Comment:
maybe a more specific term for the key could be `boundary`, PointList is appropriate for the schema name